OMF_API/AppSettings.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System.Collections.Generic;
+
+namespace OMF_API
+{
+    public class AppSettings
+    {
+        /// <summary>
+        /// The list of endpoints to connect and send to
+        /// </summary>
+        public IList<Endpoint> Endpoints { get; set; }
+    }
+
+    public class Endpoint
+    {
+        /// <summary>
+        /// The endpoint type. This will be OCS, EDS, or PI
+        /// </summary>
+        public string EndpointType { get; set; }
```

Comment:
should this be an enum?

Reply:
An enum would be generally preferable here, but enums can only be integral values. I suppose that I could read the strings and then convert them into an enum, but I will leave this for a future update.